### ISSUE TITLE
Table with Long Cells for PDF Output

### ISSUE THREAD

**Issue by sammo3182** [rstudio/gt#174]
Is there a way to add line breaks or change the font size of tables including long sentence cells or with many columns? Here attached is a minimal example of a long-sentence cell table.

[minimal_example.zip](https://github.com/rstudio/gt/files/2855210/minimal_example.zip)

**Comment by gitmoneyMSBA** [comment]
I agree that this would be a useful feature, or a wholesale shrink to fit **(kable has a "scale down" option in kable extra).** For column outputs >4 or 5 variables, I've found most output bleeds over the page and disappears.

**Comment by r-cheologist** [comment]
Is there any movement/roadmap for this? I saw  #387  with no apparent reaction byt the `gt` team ... The functionality is much needed...!
